examples/config-openid.json

```diff
@@ -33,11 +33,18 @@
             "clientsecret": "client_secret",
             "scopes": ["openid", "read_api", "read_user", "profile", "email"]
           },
-          "dex": {
+          "oidc": {
```

Comment:
@peusebiu oh! are we allowed to do this?

Reply:
If mutiple oidc backends to be supported, then this config model wont work for us.
We need a "name": { "type": <>...} model

Reply:
I agree but this would be heavier rework and generalize in zui too.

That said in lots of cases(*) you have one source of external truth (or a mix of local + one external provider). [* not all cases of courses]
Having multiple might lead to identity overlapping [depends if you use email or username as reference in your oidc], unless you add the auth source in the acl too.

Reply:
> @peusebiu oh! are we allowed to do this?

Did I forgot to remove my other example entry ?

Reply:
fixed, my bad